### ISSUE TITLE
Enable image to work with only the root disk for AWS t2 instances

### ISSUE THREAD

**Issue by slivne** [scylladb/scylla-machine-image#170]
This was working in past images

The root disk is now set to 30GB it has enough space for both the swap and the data

I tested it yesterday and found what needs to be done to make it working - in a hack way

The main issue is that we now seem to set performance setting - although the instance is t2 and should run in developer-mode

update /etc/default/scylla-server 
```
SET_CLOCKSOURCE=no
```

```
sudo scylla_setup --no-raid-setup --no-ec2-check --no-coredump-setup  --no-io-setup --no-sysconfig-setup --no-node-exporter 
sudo scylla_dev_mode_setup --developer-mode 1
sudo systemctl disable scylla-image-setup.service 
sudo systemctl start scylla-server.service 
```

I expect some of this to be not needed - I did not attempt to find the minimal set.

Once we have this working we need to make the image work also on t3 with a single disk.

**Comment by slivne** [comment]
@penberg this is the non blocking issue :)

**Comment by penberg** [comment]
@syuu1228 Let's make the `t1` instances properly supported by the AMI then.

**Comment by syuu1228** [comment]
@slivne Isn't it we decided that Scylla AMI requires at least one additional disk to run?
The official document says:
https://www.scylladb.com/download/?platform=aws#open-source
_The Scylla AMI requires at least two instance store volumes. The Scylla data directory will be formatted with XFS when the instance first boots. Scylla will fail to start if only one volume is configured._

We discussed this before (scylladb/scylla#8501), and added warning message on login prompt that describes our AMI requires additional disk (#130).
